File: EBonistika.API/Entities/Collection.cs
namespace EBonistika.API.Entities;

public class Collection
{
    public Guid Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public DateTimeOffset CreatedAt { get; set; }

    public ICollection<Series> Series { get; set; } = [];
}

File: frontend/src/types/collection.ts
export interface Collection {
  id: string;
  name: string;

  // Заполняются бэком (см. EBonistika.API/CRUD_CONTRACT.md, раздел 2).
  // Опциональны, пока бэк не расширил CollectionDto — UI деградирует мягко.
  createdAt?: string;
  seriesCount?: number;
  itemCount?: number;
}
